### DISCUSSION TITLE
[Request] - TP Link Omada  AP Controller Software

### DISCUSSION THREAD

**Discussion by mitchross** [tteck/Proxmox#453] [Ideas]
https://hub.docker.com/r/mbentley/omada-controller

https://community.tp-link.com/en/business/forum/topic/261734

**Comment by tteck** [comment]
Is this not it?
```
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/omada-v4.sh)"
```

**Reply by mitchross** [reply]
DOH! I command + F'd for tplink not omada. Thanks!
